tst/EnergyPlus/unit/ChillerExhaustAbsorption.unit.cc

```diff
@@ -1155,3 +1156,280 @@ TEST_F(EnergyPlusFixture, ExhAbsorption_GetInput_Multiple_Objects_Test)
+        "    50.0,                    !- Maximum Value of x                                                            ",
+        "    0.0,                     !- Minimum Value of y                                                            ",
+        "    3050.,                   !- Maximum Value of y                                                            ",
+        "    ,                        !- Minimum Curve Output                                                          ",
+        "    ,                        !- Maximum Curve Output                                                          ",
+        "    Temperature,             !- Input Unit Type for X                                                         ",
+        "    Distance,                !- Input Unit Type for Y                                                         ",
+        "    Dimensionless;           !- Output Unit Type                                                              ",
+
+    });
+
+    ASSERT_TRUE(process_idf(idf_objects));
+    compare_err_stream("");
+
+    GetExhaustAbsorberInput(*state);
+
+    auto &thisChillerHeater = state->dataChillerExhaustAbsorption->ExhaustAbsorber(1);
+
+    Real64 loadinput = -5000.0;
+    bool runflaginput = true;
+
+    thisChillerHeater.CoolingLoad = 100000.0;
+    thisChillerHeater.CoolPartLoadRatio = 1.0;
+    state->dataPlnt->TotNumLoops = 1;
+    state->dataPlnt->PlantLoop.allocate(state->dataPlnt->TotNumLoops);
+
+    thisChillerHeater.CWPlantLoc.loopNum = 1;
+    thisChillerHeater.CWPlantLoc.loopSideNum = DataPlant::LoopSideLocation::Demand;
+
+    state->dataPlnt->PlantLoop(1).FluidName = "WATER";
+    state->dataPlnt->PlantLoop(1).FluidIndex = 1;
+    state->dataPlnt->PlantLoop(1).LoopDemandCalcScheme = DataPlant::LoopDemandCalcScheme::SingleSetPoint;
+    state->dataPlnt->PlantLoop(1).LoopSide(DataPlant::LoopSideLocation::Demand).FlowLock = DataPlant::FlowLock::Locked;
+    state->dataLoopNodes->Node(3).Temp = 60.0;
+    state->dataLoopNodes->Node(3).MassFlowRate = 0.5;
+    state->dataLoopNodes->Node(4).TempSetPoint = 70.0;
+    state->dataLoopNodes->Node(7).Temp = 350.0;
+    state->dataLoopNodes->Node(7).MassFlowRate = 0.5;
+
+    thisChillerHeater.InCoolingMode = true;
+    thisChillerHeater.isWaterCooled = true;
+
+    thisChillerHeater.calcChiller(*state, loadinput);
+
+    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 3);
+    EXPECT_EQ(state->dataErrTracking->LastSevereError,
+              "CalcExhaustAbsorberChillerModel: Condenser flow = 0, for Exhaust Absorber Chiller=EXH CHILLER");
+
+    thisChillerHeater.calcChiller(*state, loadinput);
+    EXPECT_EQ(state->dataErrTracking->TotalSevereErrors, 5);
+    EXPECT_EQ(state->dataErrTracking->LastSevereError,
+              "SetComponentFlowRate: trapped plant loop index = 0, check component with inlet node named=EXH CHILLER CONDENSER INLET NODE");
```

Comment:
Also, the tests pass when the ChillerGasAbsorption files are same as develop, should we add a test for that as well?

Reply:
I would probably save a repeated unit test on that if possible, considering these two types of chillerheaters follow similar (or actually the same) error reporting logic regarding the condenser flows.

Reply:
Got it 👍🏽